FalmouthFalcons/iteration-2
# Jim ticket7

Description:
Added destroy to product test and model.

Number of Fixes:
2
Related Ticket(s):
N/A

Problem to Solve:
Delete of products from table

Proposed Changes:

Expected Behavior:
product_test will pass on model

Steps to Test Solution
1) git pull origin master
2) git fetch
3) git checkout jim-ticket7

Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods

Deploy Notes:
- Notes regarding deployment the contained body of work.
- These should note any db migrations, etc.
